src/librustc_borrowck/diagnostics.rs

```diff
@@ -390,6 +390,85 @@ fn foo(a: &mut i32) {
 ```
 "##,
 
+E0505: r##"
+You tried to move out of a value which was borrowed before.
```

Comment:
Could you rewrite this sentence in "a neutral way" please? Something like "A value was moved out while it was still borrowed".